Sources/MongoSwift/BSON/BSONValue.swift

```diff
@@ -642,6 +642,32 @@ public struct ObjectId: BSONValue, Equatable, CustomStringConvertible, Codable {
 
 }
 
+/// Extension to allow a UUID to be initialized from a Binary BSONValue.
+extension UUID {
+    // TODO: fill the rest of this out for full BSONValue conformance.
```

Comment:
let's make a ticket for that to reference here

Reply:
filed [SWIFT-295](https://jira.mongodb.org/browse/SWIFT-295)